tasks/bootstrap_local_vm/05_add_host.yml

```diff
@@ -1,6 +1,16 @@
 ---
 - name: Add host
   block:
+  - name: Open a port on firewalld
+    firewalld:
+      service: "{{ item }}"
```

Comment:
This task need to be present for successful deployment on RHV 4.4.1 latest build. Without these for some reason engine setup `       - name: Check engine VM health ` task fails in this tasks file. This was not required in past but probably needed now? Currently when I checked services that are allowed in FirewallD on the fresh Engine were 
```
firewall-cmd --list-services
cockpit dhcpv6-client ssh
```
And Ports were blank:
```
firewall-cmd --list-ports

<No output>
```
So adding this task helped me fix the issue.